extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/lib485-bridge.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Lib485 Bridge -->
+    <bridge-type id="lib485-bridge">
+        <label>Lib485 Bridge</label>
+        <description>The Lib485 bridge represents a single DMX universe connected via Lib485</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of bridge, format: address[:port]. Default port is 9020.</description>
+                <required>false</required>
```

Comment:
I think in this case it would be better to have the parameter "required" and specify a `<default>localhost</default>`, because this is what "really happens" behind the scenes.